backend/accounts/baseaccount.go

```diff
@@ -70,14 +70,23 @@ type BaseAccount struct {
 	// synced indicates whether the account has loaded and finished the initial sync of the
 	// addresses.
 	synced  atomic.Bool
-	offline error
+	offline atomic.Value
```

Comment:
Maybe better to use `atomic.Pointer[error]`? Generic and type-safe, no need for type-casting.

Reply:
Good idea, done